approval_polls/views.py

```diff
@@ -551,7 +534,8 @@ def get(self, request, *args, **kwargs):
             'blank_choices': [],
             'choice_blank_error': False,
             'existing_choice_texts': {'new': {}, 'existing': {}},
-            'existing_choice_links': {'new': {}, 'existing': {}}
+            'existing_choice_links': {'new': {}, 'existing': {}},
+            'invited_emails': re.compile(r"\[(.*)\]").sub(r'\1', str(poll.invited_emails()))
```

Comment:
I think we could write this better using join instead. 
eg `','.join([str(r) for r in poll.invited_emails()])` 

It would help avoid the single quotes appear in the email invitees list and can be directly copy pasted. On another note we need to validate email javascript better to avoid these special characters in the email. I shall raise that as another minor issue.